src/components/app/UrlManager.js

```diff
@@ -23,6 +23,7 @@ import {
   getProfilesFromRawUrl,
```

Comment:
The change in this file should be unnecessary. Moreover it's not correct, because then we return to the zip file list whenever we hit "back" even if it's not appropriate:
* when we're looking at a file that's not a zip, pressing back will likely bring an error (I haven't tried)
* when we're looking at a zip file, did some changes in the view (for example a search, or changed tab), pressing back will go back to the zip file even though we want to go to the previous state instead.

So I think you can revert all changes in this file.